modules/kmm-configuring-kmmo.adoc

```diff
@@ -0,0 +1,109 @@
+[id="kmm-configuring-kmmo_{context}"]
+= Configuring the Kernel Module Management Operator
+
+In most cases, the default configuration for the Kernel Module Management (KMM) Operator does not need to be modified. However, you can modify the Operator settings to suit your environment using the following procedure.
+
+The Operator configuration is set in the `kmm-operator-manager-config` `ConfigMap` in the Operator namespace.
+
+.Procedure
+
+. To modify the settings, edit the `ConfigMap` data, using the following command:
++
+[source,terminal]
+----
+oc edit configmap -n "$namespace" kmm-operator-manager-config
+----
++
+.Example output
++
+[source,yaml]
+----
+healthProbeBindAddress: :8081
+leaderElection:
+  enabled: true
+  resourceID: kmm.sigs.x-k8s.io
+webhook:
+  disableHTTP2: true  # CVE-2023-44487
+  port: 9443
+metrics:
+  enableAuthnAuthz: true
+  disableHTTP2: true  # CVE-2023-44487
+  bindAddress: 0.0.0.0:8443
+  secureServing: true
+worker:
+  runAsUser: 0
+  seLinuxType: spc_t
+  setFirmwareClassPath: /var/lib/firmware
+----
++
+.Operator configuration parameters
+[cols="2,8",options="header"]
+|===
+|Parameter |Description
+
+| `healthProbeBindAddress`
+| Defines the address on which the operator monitors for kubelet health probes. The recommended value is `:8081`.
+
+|`leaderElection.enabled`
+|Determines whether leader election is used to ensure that only one replica of the KMM operator is running at any time. For more information, see https://kubernetes.io/docs/concepts/architecture/leases/[Leases]. The recommended value is `true`.
+
+|`leaderElection.resourceID`
+|Determines the name of the resource that leader election uses for holding the leader lock. The recommended value is `kmm.sigs.x-k8s.io`.
+
+|`webhook.disableHTTP2`
+|If `true`, disables HTTP/2 for the webhook server, as a mitigation for [CVE-2023-44487]. For more information, see link:https://access.redhat.com/security/cve/cve-2023-44487. The recommended value is `true`.
+
+|`webhook.port`
+|Defines the port on which the Operator monitors webhook requests. The recommended value is `9443`.
+
+|`metrics.enableAuthnAuthz`
+a|Determines if metrics are authenticated using `TokenReviews` and authorized using `SubjectAccessReviews` with the kube-apiserver.
+
+For authentication and authorization, the controller needs a ClusterRole with the following rules:
+
+* `apiGroups: authentication.k8s.io, resources: tokenreviews, verbs: create`
+
+* `apiGroups: authorization.k8s.io, resources: subjectaccessreviews, verbs: create`
+
+To scrape metrics, for example, using Prometheus, the client needs a `ClusterRole` with the following rule:
+
+* `nonResourceURLs: "/metrics", verbs: get`
+
+The recommended value is `true`.
+
+|`metrics.disableHTTP2`
+|If `true`, disables HTTP/2 for the metrics server as a mitigation for [CVE-2023-44487]. For more information, see https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487]. The recommended value is `true`.
```

Comment:
You might fix this like I did above, or fix the one above like this one. If you go with this route, I don't think that the first instance of CVE needs to be in brackets:

```suggestion
|If `true`, disables HTTP/2 for the metrics server as a mitigation for CVE-2023-44487. For more information, see https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487]. The recommended value is `true`.
```